reporter/html.js

```diff
@@ -844,16 +844,27 @@ export function escapeText( s ) {
 	} );
 
 	QUnit.testDone( function( details ) {
-		var testTitle, time, testItem, assertList,
+		var testTitle, time, testItem, assertList, status,
 			good, bad, testCounts, skipped, sourceName,
 			tests = id( "qunit-tests" );
 
 		if ( !tests ) {
 			return;
 		}
 
+
 		testItem = id( "qunit-test-output-" + details.testId );
 
+		if ( details.failed > 0 ) {
+			status = "failed";
+		} else {
+			status = details.skipped ? "skipped" : "passed";
+		}
+
+		if ( config.norender && config.norender === status ) {
```

Comment:
can norender be an array of values?

Reply:
added support for being able to parse comma separated string